app/posts/apply-for-teacher-training/2019-02-19-apply-august-2018.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: Apply – August 2018
 description: The Apply designs as they stood at the end of August 2018.
+date: 2019-02-19
```

Comment:
This date should probably be `2018-08-01`

Reply:
Updated all early Apply posts with dates that reflect titles.